errors_test.go

```diff
@@ -134,21 +139,22 @@ line 3
 line 4
 line 5`,
 			},
-			expected: `
-1| line1
-2| 
-3| line 2
-4| before highlighted after
- |        ~~~~~~~~~~~ 
-5| line 3
-6| 
-7| line 4
-`,
+			expected: "" +
```

Comment:
The point of the original literal string use is that examples could easily be typed and copy-pasted. I'm not keen on losing that.

Reply:
I 100% agree with you, it's much better.
My problem is that if you leave it like this, anyone using gofmt will break the test (as it happened to me) without noticing.
Basically, it removes trailing spaces, breaking the test.
Also, I didn't take the time to check but is it normal that the test breaks without the trailing spaces?

Reply:
It seems to me that the issue is the code generating the error is appending extra whitespaces. You're right: the test should pass when you trim end-of-line spaces in the test expected value.